TG-Netflix/src/components/lane/LaneHandler.jsx

```diff
@@ -1,146 +1,59 @@
-//     backdrop_path: "/zyO6j74DKMWfp5snWg6Hwo0T3Mz.jpg",
-//     title: "Rashomon",
-//     id: 548,
-//     video: "https://www.youtube.com/watch?v=Zqoyl2p8_lw",
-//   },
-//   {
-//     backdrop_path: "/ww9pokP6M6gp2Iu0o3sT5B7eN66.jpg",
-//     title: "The Horse Whisperer",
-//     id: 547,
-//     video: "https://www.youtube.com/watch?v=_nCowPQ2aa4",
-//   },
-//   {
-//     backdrop_path: "/rr7E0NoGKxvbkb89eR1GwfoYjpA.jpg",
-//     title: "Fight Club",
-//     id: 550,
-//     video: "https://www.youtube.com/watch?v=qtRKdVHc-cE",
-//   },
-//   {
-//     backdrop_path: "/v1QEIuBM1vvpvfqalahhIyXY0Cm.jpg",
-//     title: "The Poseidon Adventure",
-//     id: 551,
-//     video: "https://www.youtube.com/watch?v=dd03qev59Jo",
-//   },
-//   {
-//     backdrop_path: "/k4JIHyAXaGHwAwT7y5Skd17f0Wl.jpg",
-//     title: "Pane e tulipani",
-//     id: 552,
-//     video: "https://www.youtube.com/watch?v=PKTes-j1MUc",
-//   },
-//   {
-//     backdrop_path: "/r3xsFBD1VTUusk393bBc7SsDUJe.jpg",
-//     title: "Dogville",
-//     id: 553,
-//     video: "https://www.youtube.com/watch?v=dC3UFn5MeCg",
-//   },
-//   {
-//     backdrop_path: "/1qwXItFKqvKYyW1CwbYhxyUC8Pj.jpg",
-//     title: "The Cuckoo",
-//     id: 554,
-//     video: "https://www.youtube.com/watch?v=dow9SRj9A0g",
-//   },
-//   {
-//     backdrop_path: "/1fJSyFhvxBjyLZsBnYs4641YXu1.jpg",
-//     title: "Basquiat",
-//     id: 549,
-//     video: "https://www.youtube.com/watch?v=GsWZyvtX5tU",
-//   },
-//   {
-//     backdrop_path: "/zyO6j74DKMWfp5snWg6Hwo0T3Mz.jpg",
-//     title: "Rashomon",
-//     id: 548,
-//     video: "https://www.youtube.com/watch?v=Zqoyl2p8_lw",
-//   },
-//   {
-//     backdrop_path: "/ww9pokP6M6gp2Iu0o3sT5B7eN66.jpg",
-//     title: "The Horse Whisperer",
-//     id: 547,
-//     video: "https://www.youtube.com/watch?v=_nCowPQ2aa4",
-//   },
-//   {
-//     backdrop_path: "/rr7E0NoGKxvbkb89eR1GwfoYjpA.jpg",
-//     title: "Fight Club",
-//     id: 550,
-//     video: "https://www.youtube.com/watch?v=qtRKdVHc-cE",
-//   },
-//   {
-//     backdrop_path: "/v1QEIuBM1vvpvfqalahhIyXY0Cm.jpg",
-//     title: "The Poseidon Adventure",
-//     id: 551,
-//     video: "https://www.youtube.com/watch?v=dd03qev59Jo",
-//   },
-//   {
-//     backdrop_path: "/k4JIHyAXaGHwAwT7y5Skd17f0Wl.jpg",
-//     title: "Pane e tulipani",
-//     id: 552,
-//     video: "https://www.youtube.com/watch?v=PKTes-j1MUc",
-//   },
-//   {
-//     backdrop_path: "/r3xsFBD1VTUusk393bBc7SsDUJe.jpg",
-//     title: "Dogville",
-//     id: 553,
-//     video: "https://www.youtube.com/watch?v=dC3UFn5MeCg",
-//   },
-// ];
+  const dispatch = useDispatch();
+
+  const categoryList = [
+    "action",
+    "adventure",
+    "animation",
+    "comedy",
+    "crime",
+    "documentary",
+    "drama",
+    "family",
+    "fantasy",
+    "history",
+    "horror",
+    "music",
+    "mystery",
+    "romance",
+    "science fiction",
+    "tv movie",
+    "thriller",
+    "war",
+    "western",
+  ];
```

Comment:
De films in de 'Latest' categorie hebben jammer genoeg nauwelijks eigen afbeeldingen. Mss die categorie onderaan de lijst zetten en verwisselen met Action?